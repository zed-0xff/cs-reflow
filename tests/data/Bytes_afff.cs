public class Bytes_afff
{
	public static int _114;
	public static int _118;
	public static int _19;
	public static int _12;
	public static int _198;
	public static int _183;
	public static int _55;
	public static int _77;
	public static int _24;
	public static int _33;
	public static int _54;
	public static int _66;
	public static int _199;
	public static int _39;
	public static int _139;
	public static int _171;
	public static int _149;
	public static int _11;
	public static int _23;
	public static int _78;
	public static int _222;
	public static int _233;
	public static int _244;
	public static int _245;
	public static int _234;
	public static int _111;
	public static int _125;
	public static int _37;
	public static int _137;
	public static int _42;
	public static int _43;
	public static int _56;
	public static int _92;
	public static int _97;
	public static int _99;
	public static int _129;
	public static int _151;
	public static int _76;
	public static int _177;
	public static int _142;
	public static int _96;

	static Bytes_afff()
	{
		int num = 192;
		_114 = 114;
		num = 118;
		num = 118;
		int bzh = default(int);
		while (true)
		{
			int num2 = -444587964;
			while (true)
			{
				int num3 = num2;
				uint num4;
				int num5;
				switch ((num4 = (uint)(~((~(-(num3 - 1630731538) * 1694686725 + 1320580186) ^ -1069344729) - -623890005) * -1522597029)) % 42)
				{
				case 31u:
					break;
				case 41u:
					_198 = 198;
					num2 = ((int)num4 * -1027917574) ^ -561265476;
					continue;
				case 25u:
					_199 = 199;
					num2 = ((int)num4 * -697969470) ^ 0x4FADC62E;
					continue;
				case 34u:
					_39 = 39;
					num2 = ((int)num4 * -554963163) ^ 0xD464655;
					continue;
				case 15u:
					_137 = 137;
					num2 = (int)(num4 * 708252225) ^ -679208224;
					continue;
				case 20u:
					goto IL_016c;
				case 9u:
					goto IL_0190;
				case 29u:
					_142 = 142;
					num2 = ((int)num4 * -1996059474) ^ 0x41401226;
					continue;
				case 26u:
					goto IL_01c7;
				case 0u:
					goto IL_01e4;
				case 17u:
					_77 = 77;
					num2 = ((int)num4 * -1051830529) ^ -1630370554;
					continue;
				case 21u:
					_54 = 54;
					num2 = (int)(num4 * 2117470226) ^ -1063858327;
					continue;
				case 24u:
					goto IL_0235;
				case 2u:
					goto IL_0252;
				case 3u:
					_19 = 19;
					num2 = (int)((num4 * 253537150) ^ 0x2755026B);
					continue;
				case 38u:
				{
					_43 = 43;
					int num6 = 21;
					num5 = num6;
					goto IL_0298;
				}
				case 16u:
					_76 = 76;
					num2 = (int)(num4 * 1428363974) ^ -1112781676;
					continue;
				case 6u:
				{
					_78 = 78;
					int iend = Structs_e0d5.IEND;
					num5 = 14; //((-671088640 + (iend >>> 24) == 65536 * (bzh << 20)) ? (-725462250) : 14);
					goto IL_0298;
				}
				case 5u:
					goto IL_03e7;
				case 1u:
					goto IL_0414;
				default:
					goto IL_044e;
				case 37u:
					goto IL_045d;
				case 10u:
					_111 = 111;
					goto IL_0493;
				case 23u:
					goto IL_04a6;
				case 12u:
					_183 = 183;
					num2 = (int)((num4 * 1605632607) ^ 0x211EB2AA);
					continue;
				case 30u:
					_33 = 33;
					goto IL_04e4;
				case 35u:
					goto IL_051c;
				case 13u:
					goto IL_0536;
				case 19u:
					_66 = 66;
					num2 = (int)(num4 * 1075143149) ^ -265196364;
					continue;
				case 40u:
					goto IL_055f;
				case 22u:
					_11 = 11;
					goto IL_0580;
				case 14u:
					goto IL_0593;
				case 32u:
					_151 = 151;
					goto IL_0466;
				case 27u:
					_12 = 12;
					num2 = ((int)num4 * -2065717757) ^ 0x43E06ED0;
					continue;
				case 36u:
					goto IL_05e9;
				case 7u:
					goto IL_0603;
				case 8u:
					goto IL_061d;
				case 39u:
					goto IL_063a;
				case 18u:
					_97 = 97;
					goto IL_0401;
				case 11u:
					goto IL_0663;
				case 28u:
					_118 = 118;
					goto IL_0381;
				case 33u:
					_139 = 139;
					goto IL_05d6;
				case 4u:
					return;
					IL_0298:
					switch (num5)
					{
					case 9:
						break;
					case 12:
						goto IL_016c;
					case 5:
						goto IL_0190;
					case 28:
						goto IL_01c7;
					case 20:
						goto IL_01d1;
					case 0:
						goto IL_01e4;
					case 4:
					case 29:
						goto IL_01ee;
					case 35:
						goto IL_0235;
					case 26:
					case 38:
						goto IL_023f;
					case 19:
						goto IL_0252;
					case 48:
					case 50:
						goto IL_036e;
					case 39:
						goto IL_0381;
					case 24:
						goto IL_03e7;
					case 22:
						goto IL_0401;
					case 27:
						goto IL_0414;
					case 18:
						goto IL_041b;
					default:
						goto IL_0428;
					case 13:
						goto IL_043b;
					case 45:
						goto IL_044e;
					case 33:
						goto IL_045d;
					case 15:
						goto IL_0466;
					case 14:
						goto IL_0479;
					case 31:
						goto IL_0493;
					case 3:
						goto IL_04a6;
					case 11:
					case 16:
					case 30:
						goto IL_04ad;
					case 6:
						goto IL_04e4;
					case 40:
						num = (num & -129) | 0x36;
						return;
					case 17:
					case 21:
						goto IL_0509;
					case 32:
						goto IL_051c;
					case 44:
						goto IL_0536;
					case 25:
						goto IL_055f;
					case 46:
						goto IL_0580;
					case 49:
						goto IL_0593;
					case 7:
						goto IL_059a;
					case 41:
						goto IL_05d6;
					case 37:
						goto IL_05e9;
					case 8:
					case 23:
					case 36:
						goto IL_05f0;
					case 10:
						goto IL_0603;
					case 2:
						goto IL_061d;
					case 43:
						goto IL_063a;
					case 47:
						goto IL_0663;
					case 1:
					case 34:
						goto IL_066d;
					}
					goto case 25u;
					IL_0663:
					_171 = 171;
					goto IL_066d;
					IL_066d:
					num2 = ((int)num4 * -729279754) ^ -45098374;
					continue;
					IL_063a:
					_222 = 222;
					num2 = (int)((num4 * 967097142) ^ 0x522F8726);
					continue;
					IL_061d:
					_234 = 234;
					num2 = ((int)num4 * -1525042593) ^ 0x43F0C200;
					continue;
					IL_0603:
					_56 = 56;
					num2 = ((int)num4 * -1810922193) ^ -501825382;
					continue;
					IL_05e9:
					_125 = 125;
					goto IL_05f0;
					IL_05f0:
					num2 = (int)((num4 * 1788489224) ^ 0x35523595);
					continue;
					IL_05d6:
					num2 = ((int)num4 * -999037785) ^ -1443403172;
					continue;
					IL_0593:
					_55 = 55;
					goto IL_059a;
					IL_059a:
					num2 = ((int)num4 * -772262390) ^ -741873697;
					continue;
					IL_0580:
					num2 = ((int)num4 * -294531072) ^ -1293856743;
					continue;
					IL_055f:
					_96 = 96;
					num2 = ((int)num4 * -1450011376) ^ -335398316;
					continue;
					IL_0536:
					_244 = 244;
					goto IL_036e;
					IL_051c:
					_24 = 24;
					num2 = (int)(num4 * 1158479763) ^ -532068057;
					continue;
					IL_0509:
					num2 = (int)(num4 * 1989130708) ^ -955181269;
					continue;
					IL_04e4:
					num2 = (int)(num4 * 242668624) ^ -1842858691;
					continue;
					IL_04a6:
					_92 = 92;
					goto IL_04ad;
					IL_04ad:
					num2 = (int)((num4 * 19878492) ^ 0x6FA72BEA);
					continue;
					IL_0493:
					num2 = (int)((num4 * 1572074281) ^ 0x544E20C8);
					continue;
					IL_0479:
					num2 = (int)(num4 * 153208312) ^ -1019357347;
					continue;
					IL_0466:
					num2 = ((int)num4 * -410426042) ^ 0x1F82233E;
					continue;
					IL_045d:
					_99 = 99;
					goto IL_043b;
					IL_044e:
					num5 = 40;
					goto IL_0298;
					IL_043b:
					num2 = ((int)num4 * -1956340287) ^ 0x4DCD2E95;
					continue;
					IL_0414:
					_42 = 42;
					goto IL_041b;
					IL_041b:
					num = (num & -129) | 0x36;
					goto IL_0428;
					IL_0428:
					num2 = ((int)num4 * -395105061) ^ 0x60E8AEC1;
					continue;
					IL_0401:
					num2 = ((int)num4 * -851736913) ^ 0x630D8BE9;
					continue;
					IL_03e7:
					_23 = 23;
					num2 = (int)(num4 * 189948482) ^ -1245863432;
					continue;
					IL_0381:
					num2 = ((int)num4 * -621726426) ^ -1073030455;
					continue;
					IL_036e:
					num2 = ((int)num4 * -562213391) ^ 0x85B9551;
					continue;
					IL_0252:
					_129 = 129;
					num2 = (int)((num4 * 816004590) ^ 0x33C6E0E6);
					continue;
					IL_0235:
					_233 = 233;
					goto IL_023f;
					IL_023f:
					num2 = (int)(num4 * 723058973) ^ -991827339;
					continue;
					IL_01e4:
					_149 = 149;
					goto IL_01ee;
					IL_01ee:
					num2 = (int)((num4 * 69245370) ^ 0x18285C0A);
					continue;
					IL_01c7:
					_177 = 177;
					goto IL_01d1;
					IL_01d1:
					num2 = ((int)num4 * -194102847) ^ -1262221655;
					continue;
					IL_0190:
					_37 = 37;
					num2 = (int)(num4 * 1107497100) ^ -1806514581;
					continue;
					IL_016c:
					_245 = 245;
					bzh = Structs_a7cb.BZh;
					num2 = (int)(num4 * 1216917579) ^ -1767699836;
					continue;
				}
				break;
			}
		}
	}
}
