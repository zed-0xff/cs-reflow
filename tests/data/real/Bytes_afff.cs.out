static Bytes_afff()
{
    _114 = 114;
    _118 = 118;
    _19 = 19;
    _12 = 12;
    _198 = 198;
    _183 = 183;
    _55 = 55;
    _77 = 77;
    _24 = 24;
    _33 = 33;
    _54 = 54;
    _66 = 66;
    _199 = 199;
    _39 = 39;
    _139 = 139;
    _171 = 171;
    _149 = 149;
    _11 = 11;
    _23 = 23;
    _78 = 78;
    _222 = 222;
    _233 = 233;
    _244 = 244;
    _245 = 245;
    _234 = 234;
    _111 = 111;
    _125 = 125;
    _37 = 37;
    _137 = 137;
    _42 = 42;
    _43 = 43;
    _56 = 56;
    _92 = 92;
    _97 = 97;
    _99 = 99;
    _129 = 129;
    _151 = 151;
    _76 = 76;
    _177 = 177;
    _142 = 142;
    _96 = 96;
}

